src/OpenLoco/src/OpenLoco.cpp

```diff
@@ -281,17 +281,18 @@ namespace OpenLoco
         std::srand(std::time(nullptr));
         addr<0x0050C18C, int32_t>() = addr<0x00525348, int32_t>();
         call(0x004078BE); // getSystemTime unused dead code?
+
         World::TileManager::allocateMapElements();
         Environment::resolvePaths();
         Localisation::enumerateLanguages();
         Localisation::loadLanguageFile();
+
         Ui::ProgressBar::begin(StringIds::loading);
         Ui::ProgressBar::setProgress(30);
         startupChecks();
         Ui::ProgressBar::setProgress(40);
         Ui::ProgressBar::end();
```

Comment:
To do: get rid of the progress bar here? It's over in a flash, due to this `startupChecks` no longer doing any checksums.

Reply:
Is there any harm in keeping it? People might have systems that are slower?

Reply:
We are talking micro seconds on the slowest machines

Reply:
no problem then, happy to remove it